metadata_service/proxy/atlas_proxy.py

```diff
@@ -411,6 +411,8 @@ def get_table(self, *, table_uri: str) -> Table:
 
             reports_guids = [report.get("guid") for report in attrs.get("reports") or list()]
 
+            is_view = True if attrs.get('tableType', '').lower().find('view') > -1 else False
```

Comment:
minor nit: Could you please modify this check to say `!= -1` instead of `> -1`? It wasn't immediately apparent (atleast to me) why we are doing > -1